server/neptune/workflows/internal/terraform/workflow.go

```diff
@@ -45,22 +45,13 @@ const (
 )
 
 const (
-	PlanReviewSignalName   = "planreview"
-	ScheduleToCloseTimeout = 30 * time.Minute
-	HeartBeatTimeout       = 1 * time.Minute
-	TerraformClientError   = "TerraformClientError"
+	PlanReviewSignalName     = "planreview"
+	ScheduleToCloseTimeout   = 30 * time.Minute
+	HeartBeatTimeout         = 1 * time.Minute
+	TerraformClientErrorName = "TerraformClientError"
 )
 
 func Workflow(ctx workflow.Context, request Request) error {
-	options := workflow.ActivityOptions{
```

Comment:
Without this, I believe we go back to the defaults for the first activity of the workflow.

Reply:
Fixed this in the latest implementation.